bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifier.scala

```diff
@@ -59,6 +65,7 @@ trait BagVerifier[BagContext <: BagVerifyContext[BagPrefix], BagLocation <: Loca
     bag: Bag
   ): Either[BagVerifierError, Unit]
 
+  def isRetriable(error: BagVerifierError): Boolean
```

Comment:
nit: both “retriable” and “retryable” are correct spellings; elsewhere in the scala-libs we picked “retryable”. Could you change to be consistent?

And also add a newline to separate this from the comment.